swagger.yaml

```diff
@@ -6204,6 +6252,11 @@ definitions:
     type: object
     description: List of IP address pools for services.
     properties:
+      cluster_id:
+        type: string
+        format: uuid
+        description: The cluster that this network is associated with.
+        x-go-custom-tag: gorm:"primary_key;foreignkey:Cluster"
```

Comment:
If there is a primary key only for cluster-id, then there will be only a single record per cluster in this table - which is not what we want.